data-canary/scripts/talkactions/player/server_info.lua

```diff
@@ -25,14 +25,14 @@ function serverInfo.onSay(player, words, param)
 		.. "\nSpawns rate: " .. configManager.getNumber(configKeys.RATE_SPAWN) .. "x"
 	end
 		text = text .. "\n\nMore Server Info: \n"
-		.. "\nLevel to buy house " .. configManager.getNumber(configKeys.HOUSE_BUY_LEVEL)
+		.. "\nLevel to buy house: " .. configManager.getNumber(configKeys.HOUSE_BUY_LEVEL)
 		.. "\nProtection level: " .. configManager.getNumber(configKeys.PROTECTION_LEVEL)
 		.. "\nWorldType: " .. configManager.getString(configKeys.WORLD_TYPE)
 		.. "\nKills/day to red skull: " .. configManager.getNumber(configKeys.DAY_KILLS_TO_RED)
 		.. "\nKills/week to red skull: " .. configManager.getNumber(configKeys.WEEK_KILLS_TO_RED)
 		.. "\nKills/month to red skull: " .. configManager.getNumber(configKeys.MONTH_KILLS_TO_RED)
 		.. "\nServer Save: " .. configManager.getString(configKeys.GLOBAL_SERVER_SAVE_TIME)
-		player:showTextDialog(34266, text)
+	player:showTextDialog(34266, text)
```

Comment:
I know you didn't change this so it's OK, but do you know what this number means?

Reply:
From what I took a look at, the number must be the ID of the item that appears in the corner of the popup

Reply:
Yes. it's a icon of item.